plugin/status.go

```diff
@@ -0,0 +1,19 @@
+package plugin
+
+type Status struct {
+	DriverName     string
+	MeshDriverName string `json:"MeshDriverName,omitempty"`
+	Version        int
+}
+
+func NewStatus(address, meshAddress string, isPluginV2 bool) *Status {
+	status := &Status{
+		DriverName:     pluginNameFromAddress(address),
+		MeshDriverName: pluginNameFromAddress(meshAddress),
+		Version:        1,
```

Comment:
No strong opinion but maybe:
- `const PluginV1 = 1`?
- `const PluginV2 = 2`?

given we have `isPluginV2`.